test/e2e/unsupported_overrides_test.go

```diff
@@ -0,0 +1,120 @@
+/*
+				BREAKAGE NOTIFICATION TEST
+  The purpose of this test is provde a general notification if and when UnsupportedConfigOverrides
+  either become broken and/or are removed all together.  The OpenShift Starter environments utilize
+  these configurations and we'd like to be notified when this tests fails.  The following email
+  address can be used to alert us of any issues if and when they arrive:
+			openshift-cr@redhat.com
+
+  THIS TEST CAN SAFELY BE SKIPPED ONCE A NOTIFICATION HAS BEEN SENT!
+*/
```

Comment:
would rather move that above the test. Here at the top of the file everybody misses it.

Reply:
Done!